src/context/Video/history.js

```diff
@@ -0,0 +1,53 @@
+import axios from "axios";
+import { ToastMsg } from "../../components";
+
+const addToHistory = async () => {
+    try{
+        const res = await axios.post('/api/user/history',{headers:{authorization:isLoggedIn}});
+        if(res.status === 201){
+            videoDispatch({type:"ADD_TO_HISTORY",payload: res.data.history});
+        }
+    }
+    catch(error){
+        ToastMsg("Some error occured,couldn't add to history","error");
+    }
+}
+
+const deleteFromHistory = async () => {
+    try{
+        const res = await axios.delete('/api/user/history/', {headers:{authorization:isLoggedIn}});
+        if(res.status === 200){
+            videoDispatch({type: "REMOVE_FROM_HISTORY", payload: res.data.history});
+            ToastMsg("Video Removed from history","success")
+        }
+        
+    }
+    catch(error){
+
```

Comment:
You can add a toast here to show error message to the user.

Reply:
thanks for pointing it out, I have made the required changes.